plugins/importexport/pubmed/PubMedExportPlugin.inc.php

```diff
@@ -160,12 +160,22 @@ function exportIssues($issueIds, $context, $user) {
 		$pubmedExportFilters = $filterDao->getObjectsByGroup('article=>pubmed-xml');
 		assert(count($pubmedExportFilters) == 1); // Assert only a single serialization filter
 		$exportFilter = array_shift($pubmedExportFilters);
-		$submissionsIterator = Services::get('submission')->getMany([
-			'contextId' => $context->getId(),
-			'issueIds' => $issueIds,
-		]);
+
+		$input = [];
+		$sectionDao = DAORegistry::getDAO('SectionDAO');
+		foreach ($issueIds as $issueId) {
+			$sections = $sectionDao->getByIssueId($issueId);
+			$submissionsInSections = [];
+			$submissionsInSections[] = Services::get('submission')->getInSections($issueId, $context->getId());
+			foreach ($sections as $section) {
+				foreach ($submissionsInSections as $articles) {
+					$input = array_merge($input, $articles[$section->getId()]['articles']);
+				}
+			}
```

Comment:
I think this would be enough:
```
$submissionsInSections = Services::get('submission')->getInSections($issueId, $context->getId());
foreach ($sections as $section) {
$input = array_merge($input, $submissionsInSections[$section->getId()]['articles']);
}
```